netmanager/Dockerfile

```diff
@@ -10,9 +9,9 @@ COPY ./build /usr/share/nginx/html
 # expose the product on port 80
 EXPOSE 80
 
-# run
+FROM base as staging
 CMD ["nginx", "-g", "daemon off;"]
 
+FROM base as production
 
-#RUN ["npm", "install", "-g", "serve"]
-#CMD ["serve", "-s", "build”]
\ No newline at end of file
+CMD ["nginx", "-g", "daemon off;"]
```

Comment:
This looks like a big change.  How are we serving the files then @NoahNsimbe

Reply:
@ja-odur I didn't make any big change. You can compare the current staging of this file to the changes I am introducing